purchases/src/main/kotlin/com/revenuecat/purchases/common/BillingAbstract.kt

```diff
@@ -65,7 +68,7 @@ internal abstract class BillingAbstract(
     )
 
     @SuppressWarnings("LongParameterList")
-    abstract fun findPurchaseInPurchaseHistory(
+    abstract fun findPurchaseInActivePurchases(
```

Comment:
This was only being used for upgrades/downgrades. I'm not sure why were were looking in the purchase history, when upgrades/downgrades are only possible with active subs AFAIK. Now we only check for active purchases.